layouts/_default/events.html

```diff
@@ -1,113 +1,279 @@
+  {{ $title := .Title }}
+
 
+  <div
+    class="mx-4 mt-10 max-w-screen-xl items-stretch sm:flex sm:flex-wrap sm:gap-x-6 md:mt-16 lg:mx-auto lg:gap-x-8 lg:px-20"
+    @mouseover.passive.once="load()"
+    @touchstart.passive.once="load()"
+  >
+    <h2
+      class="w-full border-b border-black font-sans text-2xl font-extrabold capitalize leading-tight tracking-wide md:text-3xl"
+    >
+      Past
+      {{ $title }}
+    </h2>
 
-  <section class="bg-white px-4 pb-12 pt-6 font-serif">
-    <div class="mx-auto mb-8 mt-4 w-full max-w-screen-lg">
-      {{ partial "tw/social-buttons.html" . }}
+    <div
+      class="flex-1 justify-between py-6 sm:w-2/3"
+      x-ref="archives"
+      data-ga-category="river"
+      data-sf-campaign="701Dn000000YgodIAC"
+    >
+      <nav
+        x-data="readmore(true)"
+        x-ref="root"
+        @mouseover.once="load()"
+        @touchstart.passive.once="load()"
+      >
+        <div class="flex flex-col space-y-6 md:space-y-5" x-ref="archives">
+          {{ range $i, $page := $pages | after 5 }}
+            {{ $params := partialCached "helper/page-params" . .RelPermalink }}
+            {{ $imgOpts := dict
+              "filename" (.Param "image")
+              "credit" (.Param "image-credit")
+              "description" (.Param "image-description")
+              "link" $params.link
+              "widthRatio" 5
+              "heightRatio" 3
+              "widths" (slice 128 256 384 512 1280)
+              "figClass" "mt-1"
+              "sizeHint" "(min-width: 850px) 328px, 90vw"
+              "gravity" (.Param "image-gravity")
+            }}
+            <div data-ga-category="story-{{ add 1 $i }}">
+              {{ $eyebrow := $params.published }}
+              {{ $rubricLink := $params.link }}
+              {{ $thumbClass := $params.thumbClass }}
+              {{ partial "tw/thumb-item.html" (dict
+                "wrapperClass" "flex flex-wrap gap-x-4"
+                "thumbClass" "w-2/5 sm:w-full md:w-2/5"
+                "imgOpts" $imgOpts
+                "rubricLink" $rubricLink
+                "eyebrow" $eyebrow
+                "link" $params.link
+                "hed" .LinkTitle
+                "byline" $params.byline
+                "showDate" false
+                "showByline" $params.showByline
+                "dek" $params.blurb
+                )
+              }}
+            </div>
+          {{ end }}
+        </div>
+
+        <div class="flex py-8">
+          <div class="mx-auto w-full max-w-screen-lg">
+            {{ partial "tw/paginator-level.html" .Paginator }}
+          </div>
+        </div>
+      </nav>
     </div>
-    {{ partial "tw/list-3-col-story.html" (dict
-      "pages" $pages
-      "wrapperClass" "mb-16 mx-auto max-w-screen-lg"
-      "eyebrow" "eventDate"
-      )
-    }}
-    <div class="mx-auto max-w-screen-lg">
-      {{ partial "tw/paginator-level.html" .Paginator }}
+    <div
+      id="frontpage-sidebar"
+      x-data="funnelStatus"
```

Comment:
I don't think you're using funnelStatus anymore, so it can be removed. funnelStatus should probably be an Alpine global $variable anyway.